doc/man1/openssl-smime.pod.in

```diff
@@ -167,13 +167,13 @@ default digest algorithm for the signing key will be used (usually SHA1).
 
 =item B<-I<cipher>>
 
-The encryption algorithm to use. For example DES  (56 bits) - B<-des>,
-triple DES (168 bits) - B<-des3>,
-EVP_get_cipherbyname() function) can also be used preceded by a dash, for
-example B<-aes-128-cbc>. See L<openssl-enc(1)> for list of ciphers
-supported by your version of OpenSSL.
+The encryption algorithm to use. For example, DES (56 bits) - B<-des>,
+triple DES (168 bits) - B<-des3>, or AES (256 bits) - B<-aes256>.
```

Comment:
Should we remove the DES examples from here?

Reply:
thank you @shahsb,

you have a good point, the documentation should encourage people to use modern algorithms.

Reply:
It would be nice to remove it, but since this PR has already achieved approval it is also fine to leave as is rather than going around another approval cycle.

Reply:
I would also keep `-des3` as example for the backwards compatibility.  `-des` is useless though. But I agree with Matt, that changing this can be done as a separate PR in future.